charters/wot-ig-2019.html

```diff
@@ -167,278 +179,202 @@ <h1 id="title">Proposed Web of Things Interest Group Charter</h1>
+
+        <p>Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/2015/Process-20150901/#WGCharter">W3C Process Document</a>:</p>
+
+        <section>
           <h3 id="w3c-coordination">W3C Groups</h3>
+
           <dl>
-            <dt><a href=""><i class="todo">[other name]</i> Working Group</a></dt>
-            <dd><i class="todo">[specific nature of liaison]</i></dd>
+            <dt><a href="https://www.w3.org/WoT/WG/">Web of Things Working Group</a></dt>
+            <dd>For collaboration as outlined in <a href="#scope"></a>.</dd>
+            <dt><a href="https://www.w3.org/2018/json-ld-wg/">JSON-LD Working Group</a></dt>
+            <dd>For collaboration on JSON-LD features and WoT use cases.</dd>
+            <dt><a href="http://www.w3.org/2009/dap/">Device and Sensors Working Group</a></dt>
+            <dd>For coordination on APIs for sensors and actuators.</dd>
+            <dt><a href="https://www.w3.org/WAI/APA/">Accessible Platform Architectures Working Group</a></dt>
+            <dd>For coordination on impacts of Web of Things technologies on accessibility to users with disabilities.</dd>
+            <dt><a href="http://www.w3.org/2014/data-shapes/wiki/Main_Page">RDF Data Shapes Working Group</a></dt>
+            <dd>For expertise on expressing structural constraints on RDF graphs in relation to domain models.</dd>
+            <dt><a href="http://www.w3.org/Privacy/">Privacy Interest Group</a></dt>
+            <dd>For collaboration on privacy considerations for the Web of Things.</dd>
+            <dt><a href="http://www.w3.org/auto/">Web and Automotive Business &amp; Working Groups</a></dt>
+            <dd>For collaboration on technologies and requirements relating to connected cars and the Web of Things.</dd>
           </dl>
-
-          <p class="issue"><b>Note:</b> Do not list horizontal groups here, only specific WGs relevant to your work.</p>
+        
+        </section>
+        <section>
 
           <h3 id="external-coordination">External Organizations</h3>
-          <dl>
-            <dt><a href=""><i class="todo">[other name]</i> Working Group</a></dt>
-            <dd><i class="todo">[specific nature of liaison]</i></dd>
-          </dl>
 
+          <p>To succeed in establishing inter-platform standards, W3C needs to coordinate with IoT alliances and standards development organizations. The following provides examples of organizations with which the Interest Group has already had significant coordination, during the period of the first charter. A <a href="https://www.w3.org/WoT/IG/wiki/How_does_W3C_compare_to_other_organizations">longer list</a> is available on the Interest Group wiki.</p>
```

Comment:
The following list provides
(using "the following" as a noun is technically correct but sounds odd)
coordination, during -> coordination during (eliminate comma splice)

Reply:
Interestingly, this was from the boilerplate text..